target_postgres/sinks.py

```diff
@@ -153,12 +153,26 @@ def bulk_insert_records(  # type: ignore[override]
         # Only one record per PK, we want to take the last one
         data_to_insert: List[Dict[str, Any]] = []
 
+        def _get_record_value(column: sa.Column, record: Dict[str, Any]) -> Any:
+            value = record.get(column.name)
+
+            if value is None:
+                return None
+
+            # apply the type's process_bind_param method if it exists
+            if hasattr(column.type.__class__, "process_bind_param") and callable(
+                getattr(column.type.__class__, "process_bind_param")
+            ):
+                value = column.type.process_bind_param(value, None)
```

Comment:
I would've expected this to be handled automatically by SQLAlchemy. What happens if you don't call `_get_record_value()`?

Reply:
removed, you're right